src/components/PostComment.jsx

```diff
@@ -0,0 +1,88 @@
+import {useEffect, useState} from 'react';
+import {useParams} from 'react-router-dom';
+import {  postComment, getCommentsById } from '../utils/api';
+import username from './LoginStatus';
+
+
+export default function PostComment() {
+    const [author, setAuthor] = useState("");
+    const [body , setBody] = useState("")
+    const {article_id} = useParams();
+    const [pass, setPass] = useState("");
+    const password = "swordfish"
+
+    const handleAuthor = (event) => {
+      const value = event.target.value;
+      setAuthor(value)
+     }
+
+    const handleBody = (event) => {
+      const value = event.target.value;
+      setBody(value)
+     }
+
+    const handlePassword = (event) => {
+      const value = event.target.value;
+      setPass(value)
+     }
+    
+  
+    const handleSubmit = (event) => {
+      event.preventDefault();
+      if (pass == password){
+      postComment(article_id, author, body).then((res) => {
+        console.log(res)
+        return res;
+      })
+    } else{
+        alert("incorrect password")
+    }
+
+    }
+      
```

Comment:
If I post successfully I need to have some visual feedback that it worked. There should be some kind of message saying it worked, the form should reset, or even I should be redirected to where the comments are visualised for that article.